build/Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
-
-ENV OPERATOR=/usr/local/bin/kogito-cloud-operator \
-    USER_UID=1001 \
-    USER_NAME=kogito-cloud-operator
+FROM registry.access.redhat.com/ubi8-minimal
```

Comment:
please change to registry.redhat.io/ubi8-minimal:latest

Reply:
Thanks, @spolti . I have changed the base image.